Framework/MDAlgorithms/src/CalculateCoverageDGS.cpp

```diff
@@ -425,7 +425,9 @@ void CalculateCoverageDGS::exec() {
       std::vector<coord_t> pos(4);
       std::transform(curIntSec.getBareArray(), curIntSec.getBareArray() + 4,
                      prevIntSec.getBareArray(), pos.begin(),
-                     VectorHelper::SimpleAverage<double>());
```

Comment:
Does `SimpleAverage` produce warnings now?

Reply:
Yes because the `double` return value is being converted to a `float` (`coord_t`) in `pos`.

Reply:
Should we remove the `SimpleAverage` functor now?